.github/workflows/build_and_deploy.yaml

```diff
@@ -57,7 +57,7 @@ jobs:
           build-args: |
             SENTRY_ORG=checkmarble
             SENTRY_PROJECT=marble-frontend
-            VERSION=${{ env.VERSION }}
+            SENTRY_RELEASE=${{ env.GITHUB_SHA }}
```

Comment:
Use a default, always valid and explicit name: the commit of the release.

Initially, I tried to use the tag for prod release (so it helps when it comes to read Sentry error reports) but I'm exausted with CI fixes... It's good enough like that